grafana-plugin/src/utils/authorization/authorization.ts

```diff
@@ -112,7 +110,7 @@ export const generateMissingPermissionMessage = (permission: UserAction): string
   `You are missing the ${determineRequiredAuthString(permission)}`;
 
 export const generatePermissionString = (resource: Resource, action: Action, includePrefix: boolean): string =>
-  `${includePrefix ? `${ONCALL_PERMISSION_PREFIX}.` : ''}${resource}:${action}`;
+  `${includePrefix ? `${PluginId.OnCall}.` : ''}${resource}:${action}`;
```

Comment:
hmm should this be set to the value of `getPluginId`? For `grafana-irm-app` we'll need to be using `grafana-irm-app` prefixed RBAC permissions

Reply:
I was getting errors from engine, seems there we still expect `grafana-oncall-app` prefix, we can look into it together

Reply:
One error was for sure related to filtering users by notification permission - the schedules e2e tests were failing because of that

Reply:
ahh right, it's because https://github.com/grafana/oncall/pull/4784 needs to be merged to `dev` first

Reply:
@mderynck you and I had discussed this PR, are you opposed to merging it to `dev`?

Reply:
@joeyorlando in this case I suggest that we merge my PR as is (hardcoding `oncall` here) and I'll push small commit to your branch that reverts it to use dynamic `getPluginId()`

Reply:
sounds good 👍 (@mderynck suggested waiting a bit to merge #4784 until the plugin init work is merged/finalized)